packages/snap-controller/src/Search/README.md

```diff
@@ -136,6 +136,51 @@ const searchConfig = {
 };
 ```
 
+
+### Pagination Settings 
+
+#### `pageSizeOptions` property
+The optional `pageSizeOptions` property gives the ability to overright the default pageSizeOptions in the searchPaginationStore. These are typically used in a `<select>` dropdown to change the number of results displayed per page. You can pass in as many options as you need. Each option has a `label` and a numeric `value`. The SearchPaginationStore then automatically populates and updates the `url` and `active` values. 
+
+`label` - label text to display
+
+`value` - number of results for this selection
+
+`url` - reference to the URLManager containing href and onclick values
+
+`active` - boolean stating if current page size matches the value of this option
+
+
+```typescript
+const searchConfig = {
+	id: 'search',
+	globals: {
+		pagination: {
+			pageSize: 12
+		}
+	},
```

Comment:
For this example, I think a pageSizeOption of 12 would make sense... but... doesn't this global `pageSize` overwrite the pageSizeOption selected? Not sure it should be here in the documentation if it does...